plugins/outputs/azure_data_explorer/azure_data_explorer.go

```diff
@@ -115,28 +115,28 @@ func (adx *AzureDataExplorer) Write(metrics []telegraf.Metric) error {
 }
 
 func (adx *AzureDataExplorer) writeTablePerMetric(metrics []telegraf.Metric) error {
-	metricsPerNamespace := make(map[string][]byte)
+	tableMetricsMapping := make(map[string][]byte)
```

Comment:
I think I'd avoid using the word Mapping in the name as it might confuse with ADX's (Azure Data Explorer) concept of mapping.
Maybe we can use `tableMetricGroups`?